docs/asciidoc/static/configuration.asciidoc

```diff
@@ -240,24 +240,42 @@ input { # comments can appear at the end of a line, too
 }
 ----------------------------------
 
-[[logstash-config-field-references]]
 [float]
-=== Field References
+[[event-dependent-configuration]]
+=== Event Dependent Configuration
+
+The logstash agent is a processing pipeline with 3 stages: inputs -> filters ->
+outputs. Inputs generate events, filters modify them, outputs ship them
+elsewhere.
+
+All events have properties. For example, an apache access log would have things
+like status code (200, 404), request path ("/", "index.html"), HTTP verb
+(GET, POST), client IP address, etc. Logstash calls these properties "fields."
+
+Some of the configuration options in Logstash require the existence of fields in
+order to function.  Because inputs generate events, there are no fields to
+evaluate within the input block--they do not exist yet!
 
-All events have properties. For example, an apache access log has things
-like status code (200, 404), request path ("/", "index.html"), HTTP verb (GET, POST),
-and client IP address. Logstash calls these properties fields.
+Because of their dependency on events and fields, the following configuration
+options will only work within filter and output blocks.
+
+IMPORTANT: Field references, sprintf format and conditionals will not work in
```

Comment:
Change to: IMPORTANT: Field references, sprintf format and conditionals described below will not work in
an input block.